homeassistant/components/webhook/__init__.py

```diff
@@ -157,9 +195,10 @@ async def _handle(self, request: Request, webhook_id: str) -> Response:
         hass = request.app["hass"]
         return await async_handle_webhook(hass, webhook_id, request)
 
-    head = _handle
-    post = _handle
-    put = _handle
+
+for method in SUPPORTED_METHODS:
+    # pylint: disable=protected-access
+    setattr(WebhookView, method.lower(), WebhookView._handle)
```

Comment:
I don't think this is an improvement. Let's keep it simple:

```suggestion
    head = _handle
    post = _handle
    put = _handle
```

Reply:
Sounds good. I used your suggestion in [6be71be](https://github.com/home-assistant/core/pull/66494/commits/6be71be26ea704fb703b4c23236f5a6fd16e15ea) and added `get`.